app/assets/stylesheets/common/base/modal.scss

```diff
@@ -134,7 +134,7 @@
   }
 
   &:not(.history-modal) {
-    .modal-body:not(.reorder-categories):not(.poll-ui-builder):not(.poll-breakdown) {
+    .modal-body:not(.reorder-categories):not(.poll-ui-builder):not(.poll-breakdown):not(.sidebar-categories-form-modal) {
```

Comment:
We need to exclude this from setting `max-height: 80vh` to prevent the modal from resizing as stuff is being filtered.